packages/backend/src/graphql/schema.graphql

```diff
@@ -406,6 +406,8 @@ type Asset implements Model {
   liquidity: UInt64
   "Minimum amount of liquidity that can be withdrawn from the asset"
   withdrawalThreshold: UInt64
+  "The fee structure for the asset"
+  fee: Fee
```

Comment:
Think about this a bit, since we can have both RECEIVING and SENDING fees on this, we should have a `receivingFee` and `sendingFee` resolver instead of just `fee`

Reply:
changed to `receivingFee` and `sendingFee`